cmd/utils/revert_encryption.go

```diff
@@ -0,0 +1,63 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"github.com/frain-dev/convoy/config"
+	"github.com/frain-dev/convoy/database/postgres"
+	"github.com/frain-dev/convoy/internal/pkg/cli"
+	"github.com/frain-dev/convoy/internal/pkg/keys"
+	"github.com/frain-dev/convoy/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+func AddRevertEncryptionCommand(a *cli.App) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "revert-encryption <encryption-key>",
+		Short: "Reverts the encryption initialization for the specified table columns with the provided encryption key",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			encryptionKey := args[0]
+
+			if encryptionKey == "" {
+				return fmt.Errorf("encryption key cannot be empty")
+			}
+
+			log.Infof("Reverting encryption with the provided key...")
+
+			cfg, err := config.Get()
+			if err != nil {
+				log.WithError(err).Fatal("Error fetching the config.")
```

Comment:
We should return an error here and not panic using Fatal

Reply:
This should ideally never return an error

Reply:
True, it's a nitpick, if anything.